packages/frontend/src/scripts/transaction/submit.ts

```diff
@@ -0,0 +1,47 @@
+import { EthereumAddress } from '@explorer/types'
+
+import { signInitial } from './sign'
+import { FormState } from './types'
+
+export async function submit(state: FormState) {
```

Comment:
Outside the scope of this PR but proper states in the UI would be helpful here, e.g. loading, error, waiting for signature